kwok/README.md

```diff
@@ -20,7 +20,7 @@ Once kwok is installed and Karpenter successfully applies to the cluster, you sh
 
 ```bash
 cat <<EOF | envsubst | kubectl apply -f -
-apiVersion: karpenter.sh/v1beta1
+apiVersion: karpenter.sh/v1
```

Comment:
`expireAfter` will also need to be moved to template field

Reply:
My fault. Fixed, and I also checked other fields.

Reply:
Thanks for the change. Looks good!